components/n-ui/ads/test/oAdsConfig.spec.js

```diff
@@ -141,36 +141,80 @@ describe('Config', () => {
 			expect(config.lazyLoad.viewportMargin).to.equal('0%');
 		});
 
-		context('when screen size is less than 980px and adOptimizeLazyLoad flag is defined', () => {
-
-			beforeEach(() => {
-				sandbox.stub(utils, 'getScreenSize', () => { return 979; });
-			});
-
-			afterEach(() => {
-				sandbox.restore();
-			});
+		it('Should pass 15% when screen size is narrower than 760px and adOptimizeLazyLoad flag is defined and appName is fro', () => {
+			sandbox.stub(utils, 'getScreenSize', () => { return 750; });
```

Comment:
We should probably remove the flag from the tests...it's a little confusing otherwise

Reply:
i was planning to keep the flag but update it so that its no longer an MVT - this would allow us to toggle the viewport margin. However, i guess it doesnt give us much benefit so, as you say it might be less confusing to remove it.